draft-ietf-openpgp-pqc.md

```diff
@@ -717,6 +720,9 @@ TBD (108 for testing)  | Ed448   | 57         | 57         | 57         | 114
 
 ### ML-DSA signatures {#mldsa-signature}
 
+Throughout this specification ML-DSA refers to the pure ML-DSA version defined
```

Comment:
```suggestion
Throughout this specification ML-DSA refers to the pure version ML-DSA, i.e., in contrast to the pre-hash variant, defined
```
I think this clarification is necessary as unfortunately, as Johannes pointed out, the label "pure" is not formally defined FIPS-204.